dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/RequestParamUtils.java

```diff
@@ -153,24 +150,6 @@ private static Stream<String> parseUidString(String input) {
     return CollectionUtils.emptyIfNull(TextUtils.splitToSet(input, TextUtils.SEMICOLON)).stream();
   }
 
-  /**
-   * Validates that no org unit is present if the ou mode is ACCESSIBLE or CAPTURE. If it is, an
-   * exception will be thrown.
-   *
-   * @param orgUnits
-   * @param orgUnitMode
-   * @throws BadRequestException
-   */
-  public static void validateOrgUnitParams(
```

Comment:
I think you can rename this entire file to RequestParamsValidator as it only contains functions that validate request parameters 😄